packages/rum-core/package.json

```diff
@@ -39,5 +39,6 @@
     "error-stack-parser": "^1.3.5",
     "opentracing": "^0.14.3",
     "promise-polyfill": "^8.1.3"
-  }
+  },
+  "browserslist": ["ie 11"]
```

Comment:
webpack defaults to es6, we provide this at the repo level to bundle them appropriately for specified env.